src/libtrx/game/rooms/common.c

```diff
@@ -511,12 +512,14 @@ SECTOR *Room_GetSkySector(
 
 void Room_SetAbyssHeight(const int16_t height)
 {
-    m_AbyssHeight = height;
+    m_AbyssMinHeight = height;
+    m_AbyssMaxHeight = height == 0 ? 0 : m_AbyssMinHeight + 30 * STEP_L;
```

Comment:
Hmm, what does `30 * STEP_L` signify? Asking cause it doesn't match 16384 (16 tiles).
Also would be good to explain what is the min and max role in the calculations, as it's become quite confusing to me. I always thought it's a simple plane.